### DISCUSSION TITLE
Import my package

### DISCUSSION THREAD

**Discussion by AlonHarell** [pyodide/pyodide#2968] [Q&A]
Hey, I can't import a python package I created.
I'm almost certain I tried every way in the documentation.

The package Im trying to import is available here:
https://raw.githubusercontent.com/AlonHarell/testing_whl/main/pytry-0.1-py3-none-any.whl
It's just a simple test package, which contains a few python files and a text file (to represent data).

when I do micropip.install(url) and then micropip.list() I indeed see pytry in the list, but cant import it.
The same happens when I try to loadPackage in JS.

Does anybody have a sure, proven way to import a package?

**Comment by ryanking13** [comment]
Well, there is no python file inside the wheel file. Probably you did something wrong during packaging?

```
$ unzip -l ./pytry-0.1-py3-none-any.whl
Archive:  ./pytry-0.1-py3-none-any.whl
  Length      Date    Time    Name
---------  ---------- -----   ----
      181  2022-08-14 19:45   pytry-0.1.dist-info/METADATA
       92  2022-08-14 19:45   pytry-0.1.dist-info/WHEEL
        1  2022-08-14 19:45   pytry-0.1.dist-info/top_level.txt
      280  2022-08-14 19:45   pytry-0.1.dist-info/RECORD
---------                     -------
      554                     4 files
```

**Reply by AlonHarell** [reply]
I fixed it, but it still won't work...
Crazy thing is, when I do micropip.list(), it does find it!

My code:
```
async function main() {
  let pyodide = await loadPyodide();
  await pyodide.loadPackage("micropip");
  const micropip = pyodide.pyimport("micropip");
  await micropip.install("https://raw.githubusercontent.com/AlonHarell/testing_whl/main/pytry-0.1-py3-none-any.whl");
  pyodide.runPythonAsync(`
    import micropip
    print(micropip.list())
    import pytry

  `);
}

main();
```

but then, in the browser:
![image](https://user-images.githubusercontent.com/14362286/184680380-ffb2693c-4e16-4ba8-803c-5c84bec0681e.png)

**Reply by ryanking13** [reply]
Could you flush your browser cache and retry? I was able install and import `pytry` from given url.
